### PR TITLE
Update DCP_change_log.md

### PR DESCRIPTION
**Pull request by julienVerm**

### CODE REVIEW

**Comment by EdithLansens** [comment]
@julienVerm, the documentation about ordering credits was taken offline 3 weeks ago because the credits were not yet available on the Azure Marketplace. See: https://github.com/SkylineCommunications/dataminer-docs/commit/7a27d7853d7b7089e4fdef8a095b66a0d4014d8b. Are the credits available now? If not, we will have to hold off on merging this.

**Comment by julienVerm** [comment]
> @julienVerm, the documentation about ordering credits was taken offline 3 weeks ago because the credits were not yet available on the Azure Marketplace. See: [7a27d78](https://github.com/SkylineCommunications/dataminer-docs/commit/7a27d7853d7b7089e4fdef8a095b66a0d4014d8b). Are the credits available now? If not, we will have to hold off on merging this.

**Comment by julienVerm** [comment]
reopened as it is now live again

**Comment by MariekeGO** [comment]
@BertVandenberghe, does this mean we can add "Ordering DataMiner credits" to the TOC again?

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ The connection status information is now updated every minute on dataminer.servi
 
 In addition, a problem has been resolved that caused some characters to be invalid in the password field in the deployment form.
 
+#### 6 March 2024 - Enhancement - Ordering DataMiner credits through Azure Marketplace [ID 38909]
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:22]:
```suggestion
#### 6 March 2024 - Enhancement - Ordering DataMiner credits through Azure Marketplace [ID_38909]
```

**Review by JordyGit — APPROVED** [review]

**Comment by BertVandenberghe** [comment]
> @BertVandenberghe, does this mean we can add "Ordering DataMiner credits" to the TOC again?

ok for me

**Review by JordyGit — APPROVED** [review]

**Comment by MariekeGO** [comment]
@julienVerm, could you still update the screenshot with the project details on the [page about ordering credits](https://docs.dataminer.services/user-guide/Getting_started/DaaS/Order_DataMiner_credits.html) so that it shows a correct *Price/payment frequency* and *Subtotal*, instead of 0.00/year? 
image

A realistic example could for instance be buying 10 credits. Users are likely to look at this screenshot to know if what they're seeing is correct, so if we show €0.00 in the screenshot, this can cause confusion. Ideally, the screenshot should also use the same SaaS resource name as the screenshot below the next step in the procedure.

File: user-guide/images/Azure_Portal_Basics.png
Comment (MariekeGO) [inline user-guide/images/Azure_Portal_Basics.png:1]: Could you resize the window for this screenshot so that there is less whitespace at the bottom? It will be more user-friendly that way.

File: user-guide/Getting_started/DaaS/Order_DataMiner_credits.md [hunk 2]
```diff
@@ -27,20 +31,24 @@ To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_b
...
 1. Click *Configure account now*.
 
    This will redirect you to the DataMiner marketplace landing page.
 
    ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)
 
-1. Check whether the correct personal details are displayed, and adjust them if necessary.
-
-1. When all your personal details are correct, click *Purchase credits*.
+1. Click *Purchase credits*.
```
Comment (MariekeGO) [inline user-guide/Getting_started/DaaS/Order_DataMiner_credits.md:46]: The screenshot shows "Activate subscription", but the text says you need to click "Purchase credits". Is something missing here? Or should the text be updated?

Comment (julienVerm) [reply user-guide/Getting_started/DaaS/Order_DataMiner_credits.md:46]: I updated to hastily, the text needs to be updated

File: user-guide/images/Azure_Portal_Basics.png
Comment (MariekeGO) [inline user-guide/images/Azure_Portal_Basics.png:1]: The "Review + subscribe" button is no longer visible in this screenshot, which makes it a bit confusing. Could you take the screenshot of a sufficiently small window, so that it shows the content with the button below it, without unnecessary whitespace in between?

**Review by MariekeGO — APPROVED** [review]
